src/transaction.js

```diff
@@ -636,6 +680,40 @@ const transaction = {
   clearTransactionWeightConstants() {
     this._weightConstants = null;
   },
+
+  /**
+   * Serialize create token tx info to bytes
+   *
+   * @param {Object} txData Object with name and symbol of token
+   * {
+   *  'name': 'TokenName',
+   *  'symbol': 'TKN',
+   * }
+   *
+   * @return {Array} array of bytes
+   * @memberof Transaction
+   * @inner
+   */
+  createTokenTxInfo(txData) {
```

Comment:
The name `createTokenTxInfo` seems misleading to me, as we're not creating anything. I prefer `serializeTokenInfo` or something along those lines.

Reply:
Agreed! Thanks